kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -0,0 +1,65 @@
+### Domain lifecycle sample scripts
+The WebLogic Server Kubernetes Operator project provides a set of sample scripts to shut down or start a specific managed-server, cluster or the entire domain. These can be helpful when scripting the lifecycle of a WebLogic Server Domain.
+
+#### Scripts to start and stop a managed server
+The `startServer.sh` script starts a managed server by patching it's server start policy to `ALWAYS`. It also increases the replica count value for the managed server's cluster by `1`. The script provides an option to keep the replica count value constant. Please see script `usage` information using `-h` option.
```

Comment:
Everywhere 'server start policy' or another domain resource setting is mentioned, please prepend the words 'domain resource' and append the words 'attribute' so that the reader knows exactly what kind of attribute is being referenced - unless the doc has already clearly made a fully qualified reference to the attribute in a nearby location.  Where appropriate, it can also be helpful to provide the exact name of the attribute, eg. `spec.xxx.server-name.xxx.yyy`...

Reply:
The change here looks good, but it looks like you missed some locations - I've added new notes at each.